images.yaml

```diff
@@ -333,7 +333,7 @@
   tags:
   - sha: 2d55b56b04c06a4992729ad5430d2fae1a6833bd9c4a87a00173ff4563bbee40
     tag: 1.17-alpine
-  - sha: ae4131942b2e200dd06cbc52915b6972275ad56d1a37788661d65ead039d3cf3
+  - sha: 94da2c39b836871c1f15c49e150a5e7d9c8b24dcc5c8d6c557fee1300f2cbb0b
```

Comment:
Why are we moving tags?

Reply:
its newer. But if this is not how we do things, we can use the old sha.

Reply:
I see, it looks like upstream moves tags? That is unfortunate. Then yeah, I guess this would be the most consistent with them.

Reply:
So whats your opinion? Leave the sha alone or use the newer one? Could you approve?